CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## v0.10.0
 
-- Add support for Snapdragon 855
+- Added support for Snapdragon 855
+- Improved `VisionView` lifecycle management
+- Updated API of `VisionManager`
 
```

Comment:
Let's also add notes about fixed leaks, `Detections` rename, cones detection and new segmentation classes